airflow/contrib/operators/ssh_operator.py

```diff
@@ -86,6 +86,12 @@ def execute(self, context):
                                                             get_pty=get_pty,
                                                             timeout=self.timeout
                                                             )
+            stdin.close()
+            output=b''
```

Comment:
@OpringaoDoTurno forget to ask. Why are you not doing the same for stderr? Is that an oversight? stderr should go to log.warn

Reply:
@bolkedebruin didn't want to touch stderr as it's read when the task finishes with error:
````
exit_status = stdout.channel.recv_exit_status()
if exit_status is 0:
    ....
else:
    error_msg = stderr.read()
    raise AirflowException("error running cmd: {0}, error: {1}"
                                        .format(self.command, error_msg))
````

But now I see that if there is something written to stderr but the task completes successfully it will be ignored 😞 

I'll investigate how to achieve both stdout and stderr written to log in realtime and in the correct order...

Reply:
It is ok to have them separated. As the log at different levels it is not such a big issue. We could also think about different facilities

Reply:
I have a working version, logging stdout to log.info and stderr to log.warning, both in realtime (and in order of appearance) and accumulating them for later:
* stdout for xcom on succes if do_xcom_push
* stderr for the exception raised when exit_status <> 0

How should I proceed now: a new commit in this same branch? new JIRA + issue? other?